src/styles/index.scss

```diff
@@ -1,3 +1,38 @@
-body {
+@use './blocks/header.scss';
+@use './blocks/card.scss';
+@use './blocks/stars.scss';
+@use './utils/variables' as *;
+
+@mixin column-count($count) {
+  grid-template-columns: repeat($count, $card-width);
+}
+
+* {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  box-sizing: border-box;
+  color: $black-color;
+}
+
+.main {
+  display: grid;
+  grid-template-columns: repeat(4, $card-width);
```

Comment:
The `grid-template-columns` property is defined here directly. Consider using only the `column-count` mixin to maintain consistency and avoid redundancy.